flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/catalog/CatalogBaseTableResolutionTest.java

```diff
@@ -424,7 +424,8 @@ private static Map<String, String> catalogMaterializedTableAsProperties() throws
         properties.put("schema.3.comment", "");
         properties.put("schema.primary-key.name", "primary_constraint");
         properties.put("schema.primary-key.columns", "id");
-        properties.put("freshness", "PT30S");
+        properties.put("freshness-interval", "30");
```

Comment:
The configuration built here corresponds to the testPropertyDeSerialization test, which can also be modified.